src/components/utils/createPathIdQuery.js

```diff
@@ -0,0 +1,5 @@
+import { PATH_ID_QUERY_PARAM } from '../../consts/URLConsts';
+
+export default function createPathIdQuery(paramValue) {
+  return PATH_ID_QUERY_PARAM + paramValue;
+}
```

Comment:
As I wanted to avoid making huge underlying changes I've made the existing `Link` components from the different tutorial journey options update the `pathId` query parameters by simply appending the end of the URL with `?pathId={paramValue}` as shown by the util function here. There is a way to use the `setSearchParams` function from the v6 'react-router-dom' but that might require more changes to use something other than `Link` components so I've done it this way for now